### ISSUE TITLE
Change channel names?

### ISSUE THREAD

**Issue by richardtrinks** [Sholofly/ZiggoNext#76]
When I say Hi Google, put television on SBS 6
Then Google home indicates that it was not found.
But I say Hi google, put television on SBS 6 HD
Then it all works perfectly.
Is there a possibility that HD will be omitted?
So that we can just say television on SBS 6? Without HD suffix?
![Channel_list](https://user-images.githubusercontent.com/15097817/119172992-e1a43e00-ba66-11eb-8d51-981bb0646a69.png)

**Comment by Sholofly** [comment]
@richardtrinks Updated the component. Please update, add
```yaml
omit_channel_quality: True
```
to your config and reboot.

Please report your findings!

**Comment by richardtrinks** [comment]
Sorry I had to check things out. It works perfectly without having to name HD suffix. Perfect and thanks for that.
Found a problem, which caused the application to stop working at all. When I call Netflix or Videoland via the Google Home, it works fine, but after that the complete application no longer works. Google Home indicates to switch the television channel, but nothing happens. It also doesn't work manually via HA. It will not work again until I restart HA.
